packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -1084,18 +1114,19 @@ class _RenderListTile extends RenderBox with SlottedContainerRenderObjectMixin<_
        _titleAlignment = titleAlignment;
 
   RenderBox? get leading => childForSlot(_ListTileSlot.leading);
-  RenderBox? get title => childForSlot(_ListTileSlot.title);
+  RenderBox get title => childForSlot(_ListTileSlot.title)!;
```

Comment:
This change is kinda odd for breaking the symmetry with the other getters. Why is it guaranteed that we have a title?

Also strange in combination with line 1124 and 1129 where it is assumed that we may not have a title again...

Reply:
`_RenderListTile` assumes the title is not null in its layout methods. Also its render object widget has these fields:
```dart
  final Widget? leading;
  final Widget title;
  final Widget? subtitle;
  final Widget? trailing;
```
so unless `title` doesn't create any render objects it's going to be non-null.

 But unfortunatly `SlottedContainerRenderObjectMixin` uses `children` in its `attach` implementation so it's possible the title has not been added as a child when `attach` is called.